dangerzone/conversion/common.py

```diff
@@ -134,7 +153,7 @@ async def convert(self) -> None:
         pass
 
     def update_progress(self, text: str, *, error: bool = False) -> None:
-        if not running_on_qubes():
+        if not qubes_native_conversion():
```

Comment:
I think we should keep `running_on_qubes()` here, since we are in the sandbox environment. Granted, the result is the same, but that may change.

Reply:
Done in [6ec1b0b](https://github.com/freedomofpress/dangerzone/pull/464/commits/6ec1b0b15e6824c34a039540d4e9331773c5253c)